manifests/plugin.pp

```diff
@@ -99,7 +99,7 @@
   }
 
   $plugin_ext = regsubst($download_url, '^.*\.(hpi|jpi)$', '\1')
-  $plugin     = "${name}.${plugin_ext}"
+  $plugin     = "${name}.jpi"
```

Comment:
Wouldn't it be better to download the `jpi` by default on line 97?

Reply:
Looks like the update URL still has hpi and this is indeed the correct thing to do.